09-secrets.md

```diff
@@ -1,28 +1,33 @@
 ## Secrets top-level element
 
-Secrets are a flavour of Configs focusing on sensitive data, with specific constraint for this usage. As the platform implementation may significantly differ from Configs, dedicated Secrets section allows to configure the related resources.
+Compose provides a way for you to use secrets without having to use environment variables to store information. If you’re injecting passwords and API keys as environment variables, you risk unintentional information exposure. Environment variables are often available to all processes, and it can be difficult to track access. They can also be printed in logs when debugging errors without your knowledge. Using secrets mitigates these risks.
 
-The top-level `secrets` declaration defines or references sensitive data that can be granted to the services in this
-application. The source of the secret is either `file` or `external`.
+Secrets are also useful for locally replicating a remote development setup with Kubernetes. Using secrets makes your application scalable.
+
+The top-level `secrets` element defines or references sensitive data that can be granted to your services. The source of the secret is set with either `file` or `external`.
```

Comment:
The source can be `file` or `environment`. `external` is not a secret source, but a way to flag a resource as "already existing"